recipes-devtools/pcie-debug/files/Makefile

```diff
@@ -0,0 +1,9 @@
+#LDFLAGS += -lreadline -lcurses
```

Comment:
remove

Reply:
Oh seems it is duplicated ... So you prefer it to be in the recipe?

Reply:
Hm... it's up to you, but it seems more logical to do it in Makefile

Reply:
Hum, why is there a Makefile here at all? Make the changes in the repo (if any are needed) so it can be built without the Bitbake recipe as well?

Reply:
Yep this needs to be cleaned up! Will try to put it in the repo.
Also the Make file is not even included in the recipe :open_mouth: